controllers/updateservice_controller.go

```diff
@@ -464,6 +486,28 @@ func (r *UpdateServiceReconciler) ensurePolicyEngineService(ctx context.Context,
 	return nil
 }
 
+func validateRouteName(instance *cv1.UpdateService, name string, namespace string) error {
+	var errReasons []string
+	routeName := namePolicyEngineRoute(instance) + "-" + namespace
```

Comment:
If this check fails nothing is created so if they could try again with a shorter operand name.

Reply:
@jottofar  AFAIU this check will consider the name given to the route and then validate if the name falls within the 63 character length.  To do that it needs to take the namespace length in to account because thats how the route name is calculated. Is that right?

Reply:
Correct. Namespace is part of the route name and is out of our control and something that could change. For example, if we created the operand named `test` in namespace `openshift-update-service` the route name, before this change, would be `test-policy-engine-route-openshift-update-service` and after change would be `test-route-openshift-update-service`.

Reply:
Confirming via a 4.9.0 OpenShift Update Service operator in a `4.10.0-0.ci-2021-11-04-133306` cluster, nobody can change the name on the `UpdateService` (or other resources?):

![image](https://user-images.githubusercontent.com/209920/140415289-3c169868-0004-4610-97ea-2443cd03e32e.png)

So the repair for "the name you picked is too long" will be "notice, delete the UpdateService, and create a replacement with a shorter name".  I don't think it's worth jumping through hoops to guess at Route validity, because the upside would be shaving a minute or so off that notice cycle.  And the downside is that we need this inlined copy making assumptions about how the cluster will be generating domain names for Routes.

Reply:
I'm fine with dropping the validation piece. It's not how I originally envisioned it anyway. I'll just change this to only include the name shortening change which should reduce/eliminate issue occurrence.